helpers/bajasHelper/bajasHelper.helper.go

```diff
@@ -117,7 +215,7 @@ func TraerDatosElemento(id int) (Elemento map[string]interface{}, outputError ma
 	}
 }
 
-func GetAllSolicitudes() (historicoActa []map[string]interface{}, outputError map[string]interface{}) {
+func GetAllSolicitudes(revComite bool, revAlmacen bool) (listBajas []*models.DetalleBaja, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetAllSolicitudes hace tal cosa"
```suggestion
// GetAllSolicitudes ...
func GetAllSolicitudes(revComite bool, revAlmacen bool) (listBajas []*models.DetalleBaja, outputError map[string]interface{}) {
```